test/ContractTestContext.ts

```diff
@@ -198,7 +198,8 @@ export async function buildContractTestContext (contractRoles: ContractRoles = {
         ),
 
         context.vault.init(context.parameterStore.address, context.externalOUSD.address),
-        context.parameterStore.init(context.treasurySigner.address, context.owner.address),
+        context.parameterStore.initialize(context.owner.address),
+        context.parameterStore.changeTreasuryAddress(context.treasurySigner.address),
```

Comment:
Treasury address is no longer set in init, so need to set it up to make sure we dont break all other tests down the line that use this param from ParamStore